### ISSUE TITLE
Feature Widget Not showing lastUpdated date and time

### ISSUE THREAD

**Issue by sibaom** [capitalone/Hygieia#586]
Hi ,
I am unable to see last updated date and time in feature widget .
though its enable for other widget .

Thanks,
siba

**Comment by tabladrum** [comment]
@amp5208

**Comment by amp5208** [comment]
@tabladrum , This is, in fact, a bug.  Please mark as a bug to be fixed as soon as possible.  Label me as the assignee.

**Comment by amp5208** [comment]
**Narrative:**

As a user of Hygieia's Feature Widget, I want to be able to see when the last successful run time of the Feature collectors, so that I know how old my data is.

**Acceptance Criteria**

 - [ ] The Feature Widget displays the last run time, similar to the other widget displays

**Use Case**

```
GIVEN Hygieia's Feature Widget
WHEN A collector for the Feature Widget data successfully gathers data
THEN Hygieia's UI is updated to show the last successful run time for the Feature Widget, similar to the other widgets
```

**Comment by sibaom** [comment]
ok amp5208

**Comment by sibaom** [comment]
Any Updates on this issue @amp and @tabladrum . Is that problem with view.js in feature?
Because its not returning deferred promise .please suggest once you fix this issue.:)

**Comment by sibaom** [comment]
Issue Resolved by me :) @amp5208 @tabladrum  just need to modify the view.js load method in feature widget.

**Comment by tabladrum** [comment]
👍 
Want to do the PR?

**Comment by amp5208** [comment]
@sibaom , Thank you for resolving this!  Please submit the PR and link it to this conversation so that I can also review the change.  I am curious to know exactly what was wrong with the load method.  Also note, with the following PR (#606), I have made drastic changes to the load for the Feature widget as it is - please ensure your change works with that PR.

**Comment by sibaom** [comment]
ctrl.load = function() {
			
			var deferred = $q.defer();
			featureData.total($scope.widgetConfig.componentId, filterTeamId).then(function(data) {
				
				processTotalResponse(data);
				deferred.resolve(data.lastUpdated);
			});
			featureData.wip($scope.widgetConfig.componentId, filterTeamId).then(function(data) {
			
				processWipResponse(data);
				deferred.resolve(data.lastUpdated);
			});
			featureData.done($scope.widgetConfig.componentId, filterTeamId).then(function(data) {
			
				processDoneResponse(data);
				deferred.resolve(data.lastUpdated);
			});
			featureData.featureWip($scope.widgetConfig.componentId,filterTeamId).then(function(data) {
			
				processFeatureWipResponse(data);
				deferred.resolve(data.lastUpdated);
			});
			featureData.sprint($scope.widgetConfig.componentId, filterTeamId).then(function(data) {
			
				processSprintResponse(data);
				deferred.resolve(data.lastUpdated);
			});
			return deferred.promise;
		};

**Comment by sibaom** [comment]
@tabladrum  this is the change that I made

**Comment by sibaom** [comment]
@tabladrum  this is the change that I have made .

**Comment by amp5208** [comment]
@sibaom, thank you for including this change.  However, this will certainly conflict with a large feature that I have submitted as a PR on #606 .  Please submit your PR after #606 is resolved and on Hygieia master.

**Comment by tabladrum** [comment]
@sibaom are you planning to submit a PR?

**Comment by nireeshT** [comment]
Fixed in https://github.com/capitalone/Hygieia/pull/1013
